app/javascript/leaflet/viewers/map.js

```diff
@@ -0,0 +1,69 @@
+import { layerGroup, LatLngBounds, polygon, geoJSON, map } from "leaflet";
+import basemaps from "../basemaps.js";
+import { geoJSONToBounds } from "../utils.js";
+import LeafletViewerBase from "./base.js";
+
+export default class LeafletViewerMap extends LeafletViewerBase {
+  constructor(element, options) {
+    super(element);
+
+    this.options = {
+      // Initial bounds of map
+      bbox: [
+        [-82, -144],
+        [77, 161],
+      ],
+      opacity: 0.75,
+      ...options,
+    };
+
+    this.overlay = layerGroup();
+    this.onLoad();
+  }
+
+  onLoad() {
+    if (this.data.mapGeom) {
+      this.options.bbox = geoJSONToBounds(JSON.parse(this.data.mapGeom));
```

Comment:
everything going into geoJSONToBounds seems to need JSON.parse. I am going to suggest we have geoJSONToBounds check the type of object going into it and do a JSON.parse if it is a string.